### PR TITLE
Add NVIDIA CI periodics on 4.14 from new repo

### PR DESCRIPTION
**Pull request by empovit**
Add new NVIDIA CI image for 4.14 with operator 23.6 and 23.9,
and remove those versions from the existing CI jobs.

Buld and push an image after a RP has been successfully merged
into the nvidia-ci repo.

### CODE REVIEW

**Comment by empovit** [comment]
/pj-rehearse pull-ci-rh-ecosystem-edge-nvidia-ci-main-4.14-images
/pj-rehearse pull-ci-rh-ecosystem-edge-nvidia-ci-main-images
/pj-rehearse periodic-ci-rh-ecosystem-edge-nvidia-ci-main-4.14-nvidia-gpu-operator-e2e-23-9-x

**Comment by empovit** [comment]
/pj-rehearse pull-ci-rh-ecosystem-edge-nvidia-ci-main-4.14-images pull-ci-rh-ecosystem-edge-nvidia-ci-main-images periodic-ci-rh-ecosystem-edge-nvidia-ci-main-4.14-nvidia-gpu-operator-e2e-23-9-x

File: ci-operator/config/rh-ecosystem-edge/nvidia-ci/rh-ecosystem-edge-nvidia-ci-main.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ build_root:
     tag: golang-1.17
 images:
 - dockerfile_path: Containerfile
-  to: nvidia-ci-tools
+  to: nvidia-ci
```
Comment (empovit) [inline ci-operator/config/rh-ecosystem-edge/nvidia-ci/rh-ecosystem-edge-nvidia-ci-main.yaml:13]: @ybettan do we want also to include `promotion`  to publish the image?

```
promotion:
  to:
  - name: latest
    namespace: nvidia-ci
```

We can also mirror it it to quay.io. WDYT?

**Comment by empovit** [comment]
/pj-rehearse periodic-ci-rh-ecosystem-edge-nvidia-ci-main-4.14-nvidia-gpu-operator-e2e-23-9-x

**Review by wabouhamad — COMMENTED** [review]
/lgtm

**Comment by empovit** [comment]
/pj-rehearse periodic-ci-rh-ecosystem-edge-nvidia-ci-main-4.14-nvidia-gpu-operator-e2e-23-9-x

**Comment by empovit** [comment]
/pj-rehearse periodic-ci-rh-ecosystem-edge-nvidia-ci-main-4.14-nvidia-gpu-operator-e2e-23-9-x

**Pull request merged**